src/global_styling/variables/_borders.ts

```diff
@@ -17,27 +17,27 @@
  * under the License.
  */
 
-import { COLOR_MODE_KEY, computed } from '../../services/theme/utils';
+import { computed } from '../../services/theme/utils';
 
 export const border = {
   widthThin: '1px',
   widthThick: '2px',
-  color: computed(['colors.lightShade'], ([lightShade]) => lightShade),
+  color: computed(([lightShade]) => lightShade, ['colors.lightShade']),
   radius: '4px',
   radiusSmall: computed(
-    [`${COLOR_MODE_KEY}.border.radius`],
-    ([euiBorderRadius]) => `calc(${euiBorderRadius} * 0.5)`
-  ),
-  thin: computed(
-    [`${COLOR_MODE_KEY}.border.widthThin`, `${COLOR_MODE_KEY}.border.color`],
-    ([widthThin, color]) => `${widthThin} solid ${color}`
-  ),
-  thick: computed(
-    [`${COLOR_MODE_KEY}.border.widthThick`, `${COLOR_MODE_KEY}.border.color`],
-    ([widthThick, color]) => `${widthThick} solid ${color}`
-  ),
-  editable: computed(
-    [`${COLOR_MODE_KEY}.border.widthThick`, `${COLOR_MODE_KEY}.border.color`],
-    ([widthThick, color]) => `${widthThick} dotted ${color}`
+    ([euiBorderRadius]) => `calc(${euiBorderRadius} * 0.5)`,
+    ['border.radius']
   ),
+  thin: computed(([widthThin, color]) => `${widthThin} solid ${color}`, [
+    'border.widthThin',
+    'border.color',
+  ]),
```

Comment:
Is this a possible different way to write this, do does the array values have to be a single key?
```suggestion
  thin: computed(([border]) => `${border.widthThin} solid ${border.color}`, ['border']),
```

Reply:
Also could we simplify further to allow the second param to be a single value as well as an array like:

```suggestion
  thin: computed((border) => `${border.widthThin} solid ${border.color}`, 'border'),
```

Reply:
> Is this a possible different way to write this, do does the array values have to be a single key?

This is similar to object spreading, and I haven't been able to make it work yet. It's on my radar, though, and I'll keep exploring options.

> Also could we simplify further to allow the second param to be a single value as well as an array like

Sure, we could allow single dependency values in addition to arrays.